### ISSUE TITLE
[Shake To Summarize][Accessibility] The summarization screen cannot be closed by dragging up the website with voice-over

### ISSUE THREAD

**Issue by data-sync-user** [mozilla-mobile/firefox-ios#28532]
## Steps to reproduce
1. Open an article that is eligible for summarization using Apple Intelligence or Mistral AI.
1. Tap the lightning bolt icon to initiate the summarization process.
1. Use VoiceOver after the summarization and attempt to select the website at the bottom in order to close the summarization screen.

### Expected behavior
The website at the bottom should be selectable with voice-over after a summarization.

### Actual behavior
The summarization screen cannot be closed by dragging up the website with voice-over.

### Device & build information
* Device: iPhone 13 Pro (iOS 26)
* Build version: v9000 (59516)
* First seen version: N/A

Notes: When the X button was selected, I tried to straight select the website with voice-over but still the summarized page was selected.

!Video Clip (2025-08-07 15_57_35).mp4|width=718,height=1556,alt="Video Clip (2025-08-07 15_57_35).mp4"!

┆Issue is synchronized with this [Jira Bug](https://mozilla-hub.atlassian.net/browse/FXIOS-13110)

**Comment by data-sync-user** [comment]
➤ Cyndi Chin commented:

Andrei Bodea How should the accessibility label read for the web view?

Tagging Alina Moldovan and Diana Andreea Barladeanu since Andrei Bodea is OOO.

**Comment by data-sync-user** [comment]
➤ Cyndi Chin commented:

The label is “Close summary”, please test and confirms that the user can now close via the web view.

**Comment by data-sync-user** [comment]
➤ Diana Andreea Barladeanu commented:

Cyndi Chin Hi, “summarizing” is announced.  Validated on v142.1 (60165), with iPad Air 11'(18.6).

Also, the title of the summary is not announced as header. Should we log a bug on this? Andres Furlan 

Thanks!

Reopening the ticket.

[^ScreenRecording_08-21-2025 15-36-10_1.MP4]

**Comment by data-sync-user** [comment]
➤ Cyndi Chin commented:

Diana Andreea Barladeanu Oops I was moving too fast last week and added the wrong label! I will fix this, also I can do the header change as well. It’s a quick one so can tag it on here.

**Comment by data-sync-user** [comment]
➤ Cyndi Chin commented:

This change has been merged with header, please validate again. Thank you!! It merged for v143 and backported to v142.1-ios26.

**Comment by data-sync-user** [comment]
➤ Diana Andreea Barladeanu commented:

Verified as fixed on v9000(60444), with iPhone 16+(26.0).

Please note that the last updates are not included in the 142.1 (60432) build. Andres Furlan Cyndi Chin 

[^ScreenRecording_08-25-2025 3-25-13 PM_1.MP4]

**Comment by data-sync-user** [comment]
➤ Andrei Bodea commented:

Verified as fixed on v144.1 (62736) with iPhone 13 Pro (iOS 26).

  * !ScreenRecording_10-10-2025 13-09-18_1.mov|width=320,alt="ScreenRecording_10-10-2025 13-09-18_1.mov"!
